plugin/controller.js

```diff
@@ -52,6 +52,8 @@
                         return next(error);
                     }
 
+                    users = users.filter(user => user.uid > 0);
```

Comment:
Is it part of compatibility issues?

Reply:
It is the other issue, /points showing a guest user with [[something]] name. Guest user has uid===0

Reply:
@kotritrona Thanks for pushing this forward. @NicolasSiver Can you please merge if all sorted?

Reply:
Is there a graceful way to handle Guest user? The filter will not work since it has a negative impact on the number of shown top users (N -1)